domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/response/Response.java

```diff
@@ -96,6 +104,10 @@ public Party getDefendant() {
         return Optional.ofNullable(statementOfTruth);
     }
 
+    public void setResponseMethod(ResponseMethod responseMethod) {
```

Comment:
do we need setResponseMethod

Reply:
It makes the code a lot simpler as we pass the abstract Response around but have 3 concrete Response classes